nginx_installer/nginx_installer.py

```diff
@@ -326,10 +326,16 @@ def _give_log_recs(self, conf_loc, name, log_filename, all_formats, nginx_versio
 
     def _get_nginx_version(self):
         '''calls out to nginx -v to get the nginx version number (eg 1.4.2)'''
-        verstring = subprocess.check_output(["nginx", "-v"], stderr=subprocess.STDOUT)
-        version = verstring.split()[2]
-        vernum = version.split("/")[1]
-        return vernum
+        try:
+            verstring = check_output(["nginx", "-v"], stderr=subprocess.STDOUT)
+            version = verstring.split()[2]
+            vernum = version.split("/")[1]
+            return vernum
+        except subprocess.CalledProcessError as e:
+            self.warn("error checking nginx version (`{}`), exit status {}".format(e.cmd, e.returncode))
+            self.warn("output:")
+            self.warn(e.output)
+            return None
```

Comment:
1.0.0 seems like a reasonable fallback - the only thing we'll _not_ suggest is the `$request_id` if they're running newer than 1.11.0)